mantle/kola/tests/util/containers.go

```diff
@@ -24,7 +24,8 @@ import (
 // GenPodmanScratchContainer creates a podman scratch container out of binaries from the host
 func GenPodmanScratchContainer(c cluster.TestCluster, m platform.Machine, name string, binnames []string) {
 	cmd := `tmpdir=$(mktemp -d); cd $tmpdir; echo -e "FROM scratch\nCOPY . /" > Dockerfile;
-	        b=$(which %s); libs=$(sudo ldd $b | grep -o /lib'[^ ]*' | sort -u);
+	                sudo rm -f /etc/ld.so.cache;
+                        b=$(which %s); libs=$(sudo ldd $b | grep -o /lib'[^ ]*' | sort -u);
```

Comment:
I think this is probably going to need a heavy dose of commenting.

Reply:
I added a long comment above as the whole block is a command.